### PR TITLE
Fix/filters wrap

### LINKED ISSUE #1836: Filters buttons wrap oddly at certain zoom levels
image

### PR DESCRIPTION
**Pull request by clari182**
- closes #1836 

https://deploy-preview-1923--staging-aiid.netlify.app/apps/discover

### CODE REVIEW

**Review by lmcnulty — CHANGES_REQUESTED** [review]

File: site/gatsby-site/src/components/discover/Filter.js [hunk 1]
```diff
@@ -7,10 +7,14 @@ import VirtualFilters from './VirtualFilters';
...
 
-function ToggleContent({ label, touched, faIcon, toggled }) {
+function ToggleContent({ label, touched, faIcon, toggled, accordion = false }) {
   return (
-    <div className="flex flex-wrap items-center justify-between w-full">
-      <div className="flex text-xs items-center">
+    <div className="flex flex-nowrap items-center justify-between w-full">
+      <div
+        className={`flex items-center whitespace-nowrap ${
+          !accordion ? 'text-[0.8vw] md:text-[1vw] lg:text-xs' : 'text-xs'
```
Comment (lmcnulty) [inline site/gatsby-site/src/components/discover/Filter.js:15]: It still overflows sometimes:

image

The text also look too small at the smallest size:

image

Comment (clari182) [reply site/gatsby-site/src/components/discover/Filter.js]: Hey Luna, can you provide the screen size you're setting on the overlapping issue? I can't reproduce it. I fixed the font size. Thanks!

Comment (lmcnulty) [reply site/gatsby-site/src/components/discover/Filter.js]: It occurs for me at 1400px with the sidebar open, but investigating further it seems not to happen on a clean profile, so this is probably fine.

**Review by lmcnulty — APPROVED** [review]

**Comment by cesarvarela** [comment]
@clari182 did the Discover tests pass in a retry? I want to make sure they pass before merging since this directly affects the Discover app.

**Comment by clari182** [comment]
Yes @cesarvarela it did!

**Pull request merged**